CMSgov/cms-carts-seds
# Fix years in question labels in section 6

### Description

Fix incorrect years in section 6 questions for 2023 forms.
I also noted some 2022 forms had the incorrect years so I adjusted those, just for local dev.
Code here is based on a combination of a [former lambda ETL](https://github.com/Enterprise-CMCS/macpro-mdct-carts/blob/main/services/database/handlers/dataCorrections/convertMoneyTypeToInteger.js) and a [script ETL](https://github.com/Enterprise-CMCS/macpro-mdct-carts/blob/main/services/database/scripts/wyoming-2022-fix.js) 

Note: DB scan explicitly only scans for 2023 and section 6. No other sections should be affected by this script. 

### Related ticket(s)


CMDCT-3112

---
### How to test

- Create your own test branch or choose an existing one (I may have already run this against main 🙈)
- Go to the deployed instance of that branch
  - Generate 2023 forms as an admin
  - Go to Section 6 in any 2023 report
  - Verify the second and third questions ask for 2022 and the fourth asks for 2022 and 2023
- Get AWS credentials from carts-dev in your terminal
- From local you can run `dynamoPrefix="YOUR BRANCH NAME" node services/database/scripts/section-6-label-year-fix.js`
- This will modify section 6 of that branch
- Refresh your browser and verify the associated questions now ask for 2023 and 2024.

---
### Author checklist


---
